iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/extensions/AbstractReferencePrioritiesTest.java

```diff
@@ -198,7 +198,7 @@ void testBadPriority() {
                 .setTerminal(lineS3S4.getTerminal1())
                 .setPriority(-2)
                 .add());
-        assertEquals("Priority should be zero or positive for ReferencePriority extension", thrown.getMessage());
+        assertEquals("Priority (-2) of terminal should be zero or positive for ReferencePriority extension", thrown.getMessage());
```

Comment:
If the message is changed (see previous comment):
```suggestion
        assertEquals("Priority (-2) of terminal (equipment LINE_S3S4) should be zero or positive for ReferencePriority extension", thrown.getMessage());
```